src/promptflow/promptflow/_sdk/entities/_flow/async_dag.py

```diff
@@ -0,0 +1,73 @@
+from os import PathLike
+from typing import Union
+
+from promptflow._constants import DEFAULT_ENCODING, FlowLanguage
+from promptflow._utils.docs import AsyncFlowDoc
+from promptflow._utils.yaml_utils import load_yaml_string
+from promptflow.exceptions import UserErrorException
+
+from .dag import Flow
+
+
+class AsyncFlow(Flow):
+    __doc__ = AsyncFlowDoc.__doc__
```

Comment:
Code editors cannot resolve this __doc__.

Reply:
then let's just copy the doc string for now